src/oom/oomd-manager.c

```diff
@@ -379,13 +403,15 @@ static int monitor_cgroup_contexts_handler(sd_event_source *s, uint64_t usec, vo
 
                 r = get_monitored_cgroup_contexts_candidates(m->monitored_swap_cgroup_contexts, &candidates);
                 if (r == -ENOMEM)
-                        return log_error_errno(r, "Failed to get monitored swap cgroup candidates");
+                        return log_error_errno(r, "Failed to get monitored swap cgroup candidates: %m");
+                if (r < 0)
+                        log_debug_errno(r, "Failed to get monitored swap cgroup candidates, ignoring: %m");
 
                 r = oomd_kill_by_swap_usage(candidates, m->dry_run);
                 if (r == -ENOMEM)
-                        return log_error_errno(r, "Failed to kill cgroup processes by swap usage");
+                        return log_error_errno(r, "Failed to kill cgroup processes by swap usage: %m");
                 if (r < 0)
-                        log_info("Failed to kill any cgroup(s) based on swap");
+                        log_info_errno(r, "Failed to kill any cgroup(s) based on swap: %m");
```

Comment:
probably should be "log_notice_errno()", i.e. this is more than informational, no? also should say ", ignoring: %m" I figure